src/select-field.tsx

```diff
@@ -117,7 +118,7 @@ export function SelectField<O extends SelectOption = SelectOption>({
                 noOptionsMessage={noOptionsMessage}
                 isClearable={isClearable}
                 menuPlacement={menuPlacement}
-                placeholder={<span />}
+                placeholder={placeholder}
```

Comment:
same as above, we'll need to always render a placeholder.  Probably doesn't need to be a span element, could be a string as above?